doc/source/install/cmake_options.rst

```diff
@@ -0,0 +1,88 @@
+.. -*- rst -*-
+
+CMake options
+=============
+
+This section describes important CMake options.
+
+To get more details about installing Groonga from the source by CMake on a specific environment,
+find the document for the specific environment from :doc:`/install`.
+
+``-G GENERATOR``
```

Comment:
I feel like it's better to add a description about a command to show all existing options but I couldn't find a good command for it.
Is `cmake -LA` a good way for it?

Reply:
`cmake -L | grep '^GRN_'`. But `cmake -L` can be used after we ran `cmake`...

Reply:
Hmm, I think we should re-consider it after this PR.